lib/src/builder/parsing/declarations_from_members.dart

```diff
@@ -403,24 +406,24 @@ List<FactoryGroup> _groupFactories(BoilerplateMembers members) {
 
 const errorStateOnly =
     'Could not find matching factory, props class, and component class in this file;'
-    ' these are required to use UiState.';
+    ' these are required to use UiState. $_ensureMatchingNames';
 
 const errorFactoryOnly = 'Could not find matching props class in this file;'
     ' this is required to declare a props map view or function component,'
-    ' and a component class is also required to declare a class-based component.';
+    ' and a component class is also required to declare a class-based component. $_ensureMatchingNames';
```

Comment:
#nit This would put the first line `If all the correct boilerplate members seem to be present, ` from `_ensureMatchingNames` beyond the line length of all the other messages. You could \n before here and with the next 5 consts.

Reply:
The parts of this string are joined using adjacent string concatenation, and actually don't contain any newlines.

For instance, this message would be:
> Could not find matching props class in this file; this is required to declare a props map view or function component, and a component class is also required to declare a class-based component. If all the correct boilerplate members seem to be present, ensure that they all have matching names (e.g., "Foo" in Foo/FooProps/FooState/FooComponent).

These don't have any line lengths since they're either outputted to the console or the analyzer plugin, which wrap text as needed.